sklearn/model_selection/tests/test_validation.py

```diff
@@ -10,52 +10,55 @@
+                                   assert_raises,
+                                   assert_raise_message,
+                                   assert_raises_regex,
+                                   assert_greater,
+                                   assert_less,
+                                   assert_array_almost_equal,
+                                   assert_array_equal,
+                                   assert_warns)
 from sklearn.utils.mocking import CheckingClassifier, MockDataFrame
 
-from sklearn.model_selection import cross_val_score
-from sklearn.model_selection import cross_val_predict
-from sklearn.model_selection import cross_validate
-from sklearn.model_selection import permutation_test_score
-from sklearn.model_selection import KFold
-from sklearn.model_selection import StratifiedKFold
-from sklearn.model_selection import LeaveOneOut
-from sklearn.model_selection import LeaveOneGroupOut
-from sklearn.model_selection import LeavePGroupsOut
-from sklearn.model_selection import GroupKFold
-from sklearn.model_selection import GroupShuffleSplit
-from sklearn.model_selection import learning_curve
-from sklearn.model_selection import validation_curve
+from sklearn.model_selection import (cross_val_score,
+                                     cross_val_predict,
+                                     cross_validate,
+                                     permutation_test_score,
+                                     KFold,
+                                     StratifiedKFold,
+                                     LeaveOneOut,
+                                     LeaveOneGroupOut,
+                                     LeavePGroupsOut,
+                                     GroupKFold,
+                                     GroupShuffleSplit,
+                                     learning_curve,
+                                     validation_curve)
 from sklearn.model_selection._validation import _check_is_permutation
 
-from sklearn.datasets import make_regression
-from sklearn.datasets import load_boston
-from sklearn.datasets import load_iris
-from sklearn.metrics import explained_variance_score
-from sklearn.metrics import make_scorer
-from sklearn.metrics import accuracy_score
-from sklearn.metrics import confusion_matrix
-from sklearn.metrics import precision_recall_fscore_support
-from sklearn.metrics import precision_score
-from sklearn.metrics import r2_score
+from sklearn.datasets import (make_regression,
+                              load_boston,
+                              load_iris,
+                              make_blobs)
+
+from sklearn.metrics import (explained_variance_score,
+                             make_scorer,
+                             accuracy_score,
+                             confusion_matrix,
+                             precision_recall_fscore_support,
+                             precision_score,
+                             r2_score)
```

Comment:
different people use different ways. The other one has slight advantages with version control, i think. But we shouldn't keep going back and forth on these, so please leave it.